menoh/composite_backend/backend/generic/operator/identity.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef MENOH_IMPL_COMPOSITE_BACKEND_BACKEND_GENERIC_OPERATOR_IDENTITY_HPP
+#define MENOH_IMPL_COMPOSITE_BACKEND_BACKEND_GENERIC_OPERATOR_IDENTITY_HPP
+
+#include <menoh/array.hpp>
+#include <menoh/composite_backend/procedure.hpp>
+
+#include <iostream>
+
+namespace menoh_impl {
+    namespace composite_backend {
+        namespace generic_backend {
+            inline procedure
+            make_identity(node const& node,
+                          std::vector<array> const& input_list,
+                          std::vector<array> const& output_list) {
+                assert(input_list.size() == 1);
+                assert(output_list.size() == 1);
+
+                auto procedure = [input = input_list.at(0),
+                                  output = output_list.at(0)]() {
+                    for(decltype(total_size(input)) i = 0;
+                        i < total_size(input); ++i) {
+                        fat(output, i) = fat(input, i);
```

Comment:
`fat` only works for `float16` type.
How about just using `menoh_dtype_size(value.dtype(), &bytes)` (or internal equivalent one) and then `memcpy`ing `total_size(value)*bytes`?
I think it is more general and maybe faster (assuming highly optimized `__builtin_memcpy` is used).

Reply:
OK